tests/main/views/test_suppliers.py

```diff
@@ -1705,9 +1705,8 @@ def test_agree_variation_fails_with_no_supplier_framework(self, live_g8_framewor
         assert response2.status_code == 404
 
     def test_agree_variation_fails_with_supplier_not_on_framework(
-            self,
-            live_g8_framework_2_variations_suppliers_not_on_framework,
-            ):
+        self, live_g8_framework_2_variations_suppliers_not_on_framework
```

Comment:
The previous tests all keep `self,` on its own line and preserve the trailing comma - might be good to do the same here too for consistency?